requirements-minimal.txt

```diff
@@ -6,7 +6,7 @@ django-bootstrap-form
 django-ipware
 django-mathfilters
 django-redis
-django>=1.10,<1.10.999
+django~=1.10
```

Comment:
It is just one more thing to hate about the awful PEP440. You can read some comments in https://github.com/pypa/pip/issues/1551. I feel it's a bad idea (like almost everything else in that terrible PEP) because it's not obvious at all what it means to anyone looking at it.

However given that it has been officially adopted, I see no reason to block you.